MewsSystems/mews-flutter
# feat!: [UX-238] Update icons library

#### Summary

- added new icons
- some icons were updated
- several icons were renamed, thus breaking change
- updated all icons story

#### Testing steps

- open `Storybook`
- check whether all icons are bundled correctly

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
